kv/label.go

```diff
@@ -0,0 +1,478 @@
+package kv
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/influxdata/influxdb"
+)
+
+var (
+	labelBucket        = []byte("labelsv1")
```

Comment:
Not `"labels/v1"` like in other refactored services?

Reply:
https://github.com/influxdata/influxdb/pull/11801#discussion_r255796007

Reply:
Gotcha!